### ISSUE TITLE
Remove namelist based configuration

### ISSUE THREAD

**Issue by bolding** [gotm-model/code#15]
Remove all namelist based configuration except for turbulence and util (used by external 3D models - notably FVCOM). Maintaining the namelists configuration will be time consuming and error-prone and will not benefit the work done on GOTM being able to generate its own configuration. In addition we will remove many 100 lines of obsolete code. Included will also be the removal of all the .xml based code and files.

**Comment by adolfstips** [comment]
Seems you forgot to say the obvious, that instead yaml configuration will be used?
As long as 'some configuration" will be possible, and only the double procedure/possibility
will be removed, this is fine for me,..

**Comment by hburchard** [comment]
I fully agree. Specifically since the yaml files are nicely and clearly commented.

**Comment by bolding** [comment]
as Jorn just pointed out:

BTW this will break GOTM-GUI compat

**Comment by jornbr** [comment]
Yes, to elaborate: GOTM-GUI would need a significant update to deal with yaml configuration and such an update is currently not planned. But in my opinion that should not stop the clean-up of GOTM. To deal with this I'd propose to point GOTM-GUI explicitly to the latest GOTM that still includes nml support (v6), likely be adding GOTM as submodule in the GOTM-GUI repository.

**Comment by bolding** [comment]
I've created the nonml branch - https://github.com/gotm-model/code/tree/nonml

Please have a look and help remove all traces of namelist.

git difftool -y ..master - might be your friend

I've grepped for namelist, _nml, nml and namlst - and most of those are removed.

The new code passes the automatic tests - except for gfortran6 (but that is a different issue).

**Comment by bolding** [comment]
no - GOTM works with YAML formatted configuration files - .https://yaml.org/

when we moved from namelists to YAML we kept support for both for a while - but in the upcoming version 7 namelists will dropped.
